src/Controller/AngularTemplatesController.php

```diff
@@ -17,4 +18,15 @@ public function beforeRender(Event $event)
     public function interface_language()
     {
     }
-}
\ No newline at end of file
+
+    public function show_all_sentences_button_text($code)
+    {
+        $this->response->header('Content-Type: application/json');
+
+        if (!LanguagesLib::languageExists($code)) {
+            return $this->response->withStatus(404);
```

Comment:
`Unknown language` is found in the search field, but selecting it will result in a 404 response because `LanguagesLib::languageExists('unknown')` is `false`.